c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +394,23 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
+  var doctype = this.urlParams_['doctype'];
+  var docid = this.urlParams_['docToAssociateId'];
+  $.get(this.apiUrl_ + '/' + doctype + '/' + docid, function(doc) {
```

Comment:
> right. Where can i get the full list of possible arguments for filtering?

https://github.com/c2corg/v6_api/blob/master/c2corg_api/views/search.py#L41-L62

> and btw can you tell me where is the full doc with steps for indexing elastic search?

Do you mean this?
https://github.com/c2corg/v6_api/tree/master/c2corg_api/scripts/migration#initialize-elasticsearch